### DISCUSSION TITLE
why cant I see enything in payloads

### DISCUSSION THREAD

**Discussion by pazza2234** [3kh0/ext-remover#467] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I got a crew shim for sh1mmer, and injected it but there still is nothing in payloads. am I supost to do something?

**Comment by 3kh0** [comment, marked as answer]
Check in utilities.

**Reply by pazza2234** [reply]
I was looking for mc chromebox, witch is not in utility
